### DISCUSSION TITLE
希望清爽首页保留B站自带的返回顶部功能

### DISCUSSION THREAD

**Discussion by japengwu** [the1812/Bilibili-Evolved#3889] [功能建议 / Ideas]
希望清爽首页可以自由选择是否保留返回顶部的功能。这个功能挺好用的，突然消失了还挺不习惯，但是又想使用清爽首页，鱼与熊掌能否全都要？

**Comment by the1812** [comment]
这有啥用吗, 清爽首页又没多长

**Reply by japengwu** [reply]
我觉得挺有必要的，用起来很舒服，个人觉得网页高度超过2倍的窗口高度的网页都应该有返回顶部的功能，毕竟也能省鼠标滚轮嘛。不过只是个人的一点小建议，因为这种很简单的小功能我自己也可以单独写个油猴脚本给加上哈哈

**Reply by japengwu** [reply]
不好意思我刚研究明白，开清爽首页要开启B站的内测模式。我不在内测模式，所以很长，很长，非常长。开启内测模式之后确实不需要返回顶部了。
